llvm/test/tools/llvm-cas/make-blob.test

```diff
@@ -19,7 +19,7 @@ RUN:   FileCheck %s -check-prefix CHECK-EMPTY -allow-empty
 RUN: llvm-cas --cas %t.cas --print-kind @%t/empty.casid |\
 RUN:   FileCheck %s -check-prefix CHECK-KIND
 CHECK-EMPTY-NOT: {{.}}
-CHECK-KIND: blob
+CHECK-KIND: node
```

Comment:
maybe rename to `make-leaf-node.test`?

Reply:
For now I'm leaving it alone, since it matches the command name `--make-blob`. I'd rather leave the command-line for some future commit, but when we update that it makes sense to rename the test.